### PR TITLE
(fix) O3-2709: wrong image placeholder used for add image in patient registration

### PR DESCRIPTION
**Pull request by psworld**
## Requirements
- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
A `broken image icon` was used as a placeholder for the image in Add Image in patient registration. A svg `palceholder.svg` was being used as placeholder. I did not found any other usage or places wher this `placeholder.svg` is referenced, so I felt free to change it. This `placeholder.svg` was only being used in the `capture-photo.component.tsx`

## Before 
image

## After
image

## Related Issue
[https://openmrs.atlassian.net/browse/O3-2709](https://openmrs.atlassian.net/browse/O3-2709)

### CODE REVIEW

File: packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.component.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ const CapturePhoto: React.FC<CapturePhotoProps> = ({ initialState, onCapturePhot
   return (
     <div style={{ display: 'flex', alignItems: 'center' }}>
       <div style={{ maxWidth: '64px' }}>
-        <img src={dataUri || initialState || placeholder} alt="Preview" style={{ width: '100%' }} />
+        <img src={dataUri || initialState || placeholder} onClick={showCam} alt="Preview" style={{ width: '100%' }} />
```
Comment (brandones) [inline packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.component.tsx:34]: If you make something clickable, it must either follow the link (`a`) contract or the button contract. See https://benmyers.dev/blog/clickable-divs/ . Or just use a `<button>` element with a CSS reset (this is usually the way to go).

**Review by brandones — CHANGES_REQUESTED** [review]

**Comment by brandones** [comment]
@psworld The checkboxes in the PR description are to inform reviewers what has been done and what has not. They are not strictly requirements (other than the first one). It is not helpful to check boxes that have nothing to do with your work, or which are untrue.

image

The second checkbox, about designs, is clearly false, as what you have implemented is different from the designs. The designs can be found [on Zeplin](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/60d59b0642e296b9bd541101). In general, do not check this box if you have not looked at the designs when doing a piece of work. So for work that involves no UI changes or for which the designs are not relevant, leave the box unchecked.

The third checkbox asks about tests. This is obviously false. So do not check the box.

File: packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.component.tsx (hunk 1 above)
Comment (psworld) [reply packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.component.tsx]: @brandones Really thankyou for the review. I didn't know about the standard practice for clickable items, thankyou for your valuable time and guidance.
I have made a new commit which incorporates the changes you suggested, kindly take a look. If more changes are required please let me know.

**Comment by psworld** [comment]
> @psworld The checkboxes in the PR description are to inform reviewers what has been done and what has not. They are not strictly requirements (other than the first one). It is not helpful to check boxes that have nothing to do with your work, or which are untrue.
> 
> image
> 
> The second checkbox, about designs, is clearly false, as what you have implemented is different from the designs. The designs can be found [on Zeplin](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/60d59b0642e296b9bd541101). In general, do not check this box if you have not looked at the designs when doing a piece of work. So for work that involves no UI changes or for which the designs are not relevant, leave the box unchecked.
> 
> The third checkbox asks about tests. This is obviously false. It wouldn't make sense to test this change. So do not check the box.

I will definitely keep these things in mind. Thankyou so much 🙏

File: packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.component.tsx [hunk 2]
```diff
@@ -30,9 +30,13 @@ const CapturePhoto: React.FC<CapturePhotoProps> = ({ initialState, onCapturePhot
 
   return (
     <div style={{ display: 'flex', alignItems: 'center' }}>
-      <div style={{ maxWidth: '64px' }}>
+      <button
+        type="button"
+        onClick={showCam}
+        style={{ border: 'none', maxWidth: '64px', padding: 0, margin: 0, background: 'none', cursor: 'pointer' }}
```
Comment (brandones) [inline packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.component.tsx:36]: This should be in a style (scss) file

**Review by brandones — CHANGES_REQUESTED** [review]
Style file

File: packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.component.tsx (hunk 2 above)
Comment (psworld) [reply packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.component.tsx]: @brandones,
I have added a new file named `capture-photo.scss` and added a class named `.buttonCssReset`

There was no existing css file for this component nor it was importing styles from any other css file. So I created a new one

**Review by brandones — APPROVED** [review]
Nice work @psworld , LGTM. Welcome to the community!

**Comment by psworld** [comment]
> Nice work @psworld , LGTM. Welcome to the community!

Thanks 🙏

**Pull request merged**